src/core/probmacro.jl

```diff
@@ -0,0 +1,13 @@
+function getexprs(expr)
```

Comment:
Is this used?

Reply:
Nope, a duplicate from a previous re-write. I will remove it.